src/animation/AnimationMixer.js

```diff
@@ -83,8 +83,10 @@ Object.assign( THREE.AnimationMixer.prototype, THREE.EventDispatcher.prototype,
 
 		var root = optionalRoot || this._root,
 			rootUuid = root.uuid,
-			clipName = ( typeof clip === 'string' ) ? clip : clip.name,
-			actionsForClip = this._actionsByClip[ clipName ];
+			clipObject = typeof clip === 'string' ? this.getClipByName( clip ) : clip,
```

Comment:
I m okay if we continue down this road, but be aware that we will need to eventually create a new per-character mixer class that makes using this global mixer easier for individual characters.  It will allow one to pre-load animations into it by name and play them back by name.  I think it can be build onto of the global mixer relatively easily -- just it wasn't my intention to have a global mixer at all, but I guess we can work with it.

We always get to the right designs eventually, although sometimes we take circuitous routes to them, and if this is how we are going to get there, I am okay with that, but just be aware that we have to get back to where there are per-character mixers of some sort.